### DISCUSSION TITLE
How to collect user's bank account information in payment sheet?

### DISCUSSION THREAD

**Discussion by Mason-Medrano** [flutter-stripe/flutter_stripe#666] [Q&A]
Hello all,

I am currently using flutter_stripe for a project. I have implemented the payment sheet to collect a user's card data, save this data for later if they choose, and then make a payment using a payment intent. I followed [https://github.com/flutter-stripe/flutter_stripe/blob/893a8449765af4d135a6737bdef1b53e5e6afa63/docs/sheet.mdx](https://github.com/flutter-stripe/flutter_stripe/blob/893a8449765af4d135a6737bdef1b53e5e6afa63/docs/sheet.mdx) to get this implemented.

Is there currently support to accept a user's bank account information, like routing number and account number, in addition to their card details within the payment sheet? Thanks!

**Comment by ZacharyHandshoe** [comment]
Same here, what's the point in having flutter_stripe when it isn't all of Stripe?

**Reply by remonh87** [reply]
The paymentsheet supports iDEAL, SEPA and Soft. Also Stripe recently launched ACH payments in beta which allows you to collect American bank accounts https://stripe.com/docs/payments/ach-debit . This is not available on Android (hopefully coming soon) yet.

@ZacharyHandshoe the paymentsheet is 100% native Stripe SDK screen we only provide a dart layer around the native sdk. . If you feel sth is missing feel free to ask Stripe for more features by filing an issue in their respective iOS and Android sdk.
